docs/dev/test-configuration-options.md

```diff
@@ -839,7 +839,7 @@ We have a three-hour maximum in place to ease the transition of new users migrat
 
 ---
 ### `commandTimeout`
-<p><small>| INTEGER |</small></p>
+<p><small>| INTEGER | <span className="sauceDBlue">Desktop Only</span> |</small></p>
```

Comment:
Can we add these labels to all desktop capabilities to make it more clear. Even though they are grouped by desktop, customers don't see it and still use it for EMUSIM/RDC and vice versa

Reply:
@wswebcreation  As per our slack conversation, this is on my to-do list :)